src/RoadRegistry.BackOffice.ProjectionHost/Program.cs

```diff
@@ -0,0 +1,262 @@
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.Runtime;
+    using Amazon.S3;
+    using Be.Vlaanderen.Basisregisters.BlobStore;
+    using Be.Vlaanderen.Basisregisters.BlobStore.Aws;
+    using Be.Vlaanderen.Basisregisters.BlobStore.IO;
+    using Be.Vlaanderen.Basisregisters.BlobStore.Sql;
+    using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Runner;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore;
+    using Configuration;
+    using Framework;
+    using Microsoft.Data.SqlClient;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using Microsoft.IO;
+    using Newtonsoft.Json;
+    using NodaTime;
+    using Projections;
+    using Schema;
+    using Serilog;
+    using Serilog.Formatting.Compact;
+    using SqlStreamStore;
+
+    public class Program
+    {
+        private static readonly Encoding WindowsAnsiEncoding = Encoding.GetEncoding(1252);
+
+        public static async Task Main(string[] args)
+        {
+            Console.WriteLine("Starting RoadRegistry.BackOffice.ProjectionHost");
+
+            AppDomain.CurrentDomain.FirstChanceException += (sender, eventArgs) =>
+                Log.Debug(eventArgs.Exception, "FirstChanceException event raised in {AppDomain}.", AppDomain.CurrentDomain.FriendlyName);
+
+            AppDomain.CurrentDomain.UnhandledException += (sender, eventArgs) =>
+                Log.Fatal((Exception)eventArgs.ExceptionObject, "Encountered a fatal exception, exiting program.");
+
+            var host = new HostBuilder()
+                .ConfigureAppConfiguration((hostContext, builder) =>
+                {
+                    Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
+
+                    if (hostContext.HostingEnvironment.IsProduction())
+                    {
+                        builder
+                            .SetBasePath(Directory.GetCurrentDirectory());
+                    }
+
+                    builder
+                        .AddJsonFile("appsettings.json", true, true)
+                        .AddJsonFile($"appsettings.{Environment.MachineName.ToLowerInvariant()}.json", true, true)
+                        .AddEnvironmentVariables()
+                        .AddCommandLine(args);
+                })
+                .ConfigureLogging((hostContext, builder) =>
+                {
+                    Serilog.Debugging.SelfLog.Enable(Console.WriteLine);
+
+                    var loggerConfiguration = new LoggerConfiguration()
+                        .ReadFrom.Configuration(hostContext.Configuration)
+                        .WriteTo.Console(new RenderedCompactJsonFormatter())
```

Comment:
Shouldnt we read this from config instead?

Reply:
done